assembly/driver/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pdi-dataservice-client-assemblies</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pdi-dataservice-driver-bundle</artifactId>
+  <version>8.1.0.0-SNAPSHOT</version>
+  <name>Pentaho Community Edition Project: ${project.artifactId}</name>
+  <description>PDI Data Services driver bundle</description>
+  <url>http://www.pentaho.com</url>
+  <dependencies>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pdi-dataservice-client-plugin</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pdi-dataservice-client-plugin-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>pentaho-kettle</groupId>
+      <artifactId>kettle-core</artifactId>
+      <version>${pdi.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>pentaho-kettle</groupId>
+      <artifactId>kettle-engine</artifactId>
+      <version>${pdi.version}</version>
```

Comment:
Aren´t `kettle-core` and `kettle-engine` defined in the `<dependencyManagement>`?

Reply:
Same as above... legacy code. Either way, they are needed for the driver bundle, and not as a general dependency for the plugin itself IMO... or am I seeing this wrong?

Reply:
If they are only needed for the driver module, it's ok to leave them here.